src/main/cpp/src/json_utils.cu

```diff
@@ -110,36 +121,14 @@ std::tuple<std::unique_ptr<cudf::column>, std::unique_ptr<rmm::device_buffer>, c
         if (not_whitespace(ch)) { break; }
       }
 
-      if (i + 3 < size &&
-          (d_str[i] == 'n' && d_str[i + 1] == 'u' && d_str[i + 2] == 'l' && d_str[i + 3] == 'l')) {
-        i += 4;
-
-        // Skip the very last whitespace characters.
-        bool is_null_literal{true};
-        for (; i < size; ++i) {
-          ch = d_str[i];
-          if (not_whitespace(ch)) {
-            is_null_literal = false;
-            break;
-          }
-        }
-
-        // The current row contains only `null` string literal and not any other non-whitespace
-        // characters. Such rows need to be masked out as null when doing concatenation.
-        if (is_null_literal) {
-          output[idx] = thrust::make_tuple(false, false);
-          return;
-        }
-      }
```

Comment:
`null` string literal is not any different from other garbage (invalid JSON) strings.

Reply:
FYI Spark integration tests all passed with this removal.